public/resources/css/module/_side-nav.scss

```diff
@@ -79,37 +79,37 @@ $sidenav-width: 240px;
     overflow: hidden;
   }
 
-  input {
-    background: $white;
-    border: none;
-    border-radius: 200px;
-    box-sizing: border-box;
-    color: $sidenav-color;
-    display: inline-block;
-    font-size: 14px;
-    height: 32px;
-    margin: 0px;
-    padding: 0px $unit;
-    width: 100%;
-
-    // PLACEHOLDER TEXT
-    &::-webkit-input-placeholder { /* Chrome/Opera/Safari */
-      color: $blue-grey-100;
-      font-size: 12px;
-    }
-    &::-moz-placeholder { /* Firefox 19+ */
-      color: $blue-grey-100;
-      font-size: 12px;
-    }
-    &:-ms-input-placeholder { /* IE 10+ */
-      color: $blue-grey-100;
-      font-size: 12px;
-    }
-    &:-moz-placeholder { /* Firefox 18- */
-      color: $blue-grey-100;
-      font-size: 12px;
-    }
-  }
+  // input[type='text'] {
+  //   background: $white;
+  //   border: none;
+  //   border-radius: 200px;
+  //   box-sizing: border-box;
+  //   color: $sidenav-color;
+  //   display: inline-block;
+  //   font-size: 14px;
+  //   height: 32px;
+  //   margin: 0px;
+  //   padding: 0px $unit;
+  //   width: 100%;
+
+  //   // PLACEHOLDER TEXT
+  //   &::-webkit-input-placeholder { /* Chrome/Opera/Safari */
+  //     color: $blue-grey-100;
+  //     font-size: 12px;
+  //   }
+  //   &::-moz-placeholder { /* Firefox 19+ */
+  //     color: $blue-grey-100;
+  //     font-size: 12px;
+  //   }
+  //   &:-ms-input-placeholder { /* IE 10+ */
+  //     color: $blue-grey-100;
+  //     font-size: 12px;
+  //   }
+  //   &:-moz-placeholder { /* Firefox 18- */
+  //     color: $blue-grey-100;
+  //     font-size: 12px;
+  //   }
+  // }
```

Comment:
if we no longer need this css we should delete them. otherwise it's unclear why they are just commented out.

if we do need them we should add a comment why this is being commented out temporarily and what action is being taken to uncomment this and by whom.